### PR TITLE
Add problem 67 solution

### PR DESCRIPTION
**Pull request by Hyftar**
This PR adds the solution the the Euler Project problem 67.

### CODE REVIEW

File: project_euler/problem_67/sol1.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+That is, 3 + 7 + 4 + 9 = 23.
+Find the maximum total from top to bottom in triangle.txt (right click and
+'Save Link/Target As...'), a 15K text file containing a triangle with
+one-hundred rows.
+'''
+from __future__ import print_function
+try:
+    raw_input          # Python 2
+    xrange
+except NameError:
```
Comment (cclauss) [inline project_euler/problem_67/sol1.py]: This repo does not support legacy Python.  Please remove __print_function__, __raw_input__, and __xrange__.

File: project_euler/problem_67/sol1.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    raw_input = input  # Python 3
+    xrange = range
+
+triangle = open('triangle.txt', 'r').readlines()
```
Comment (cclauss) [inline project_euler/problem_67/sol1.py]: This file is open()ed but never close()d.

File: project_euler/problem_67/sol1.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+a = map(lambda x: x.rstrip('\r\n').split(' '), triangle)
+a = list(map(lambda x: list(map(lambda y: int(y), x)), a))
+for i in xrange(1, len(a)):
+    for j in xrange(len(a[i])):
+        if j != len(a[i - 1]):
+            number1 = a[i - 1][j]
+        else:
+            number1 = 0
```
Comment (cclauss) [inline project_euler/problem_67/sol1.py]: Optional: consider __number1 = a[i - 1][j] if j != len(a[i - 1]) else 0__

Comment (Hyftar) [reply project_euler/problem_67/sol1.py]: I have considered it but I think it hinders readability too much

File: project_euler/problem_67/sol1.py (hunk 2 above)
Comment (Hyftar) [reply project_euler/problem_67/sol1.py]: Fixed in the new proposal.

File: project_euler/problem_67/sol1.py (hunk 1 above)
Comment (Hyftar) [reply project_euler/problem_67/sol1.py]: Removed in the new proposal.

**Comment by cclauss** [comment]
https://travis-ci.org/TheAlgorithms/Python/builds/581019718#L609

**Comment by cclauss** [comment]
See: https://github.com/TheAlgorithms/Python/blob/36684db2780d695add9bd0a4523d73496cb35664/machine_learning/random_forest_regression/random_forest_regression.py#L9

**Comment by Hyftar** [comment]
@cclauss Thanks, fixing right now.

**Pull request merged**